packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -29,6 +29,18 @@ import { AuthLogger, AuthRequest } from '../../types';
 import { subscriptionProductMetadataValidator } from '../validators';
 import { StripeHandler } from './stripe';
 
+// ALLOWED_EXPAND_RESOURCE_TYPES is a map of "types" of Stripe objects that we
+// will fetch the latest of for the webhook event, _instead of_ using the
+// object in the request payload.  Products and plans are excluded because
+// `expandResource` uses the cached lists for products and plans, but the
+// cached lists themselves are updated through webhooks.
+const BYPASS_LATEST_FETCH_TYPES = ['plan', 'price', 'product'];
```

Comment:
hmm what's the advantage in omitting it?

Reply:
It has no effect currently, so it seems confusing to list it here when we don't listen for those events or handle them.

Reply:
What would be a good way to ensure that if and when we start listening to price events that we remember to add it into the list?

Reply:
The same way that we would for any other event that we decide to start listening to? Do you want to leave it there, because at some point in the indefinite future, we will switch to using the Prices API instead of the Plans API? If you want to leave it there, can you add a comment acknowledging why it was left there?

Also you just made me realize I think we may want to add `tax_rate` to this list.

Reply:
> The same way that we would for any other event that we decide to start listening for? Do you want to leave it there, because at some point in the indefinite future, we will switch to using the Prices API instead of the Plans API? If you want to leave it there, can you add a comment explaining why it was left there?
> 
I would rather err on potentially preventing a subtle bug than potentially causing some confusion.

As for switching, I think at some point Stripe might force our hand.  Of course I don't truly know when or if we _must_ make that switch.

Sure I'll add a comment.

> Also you just made me realize I think we may want to add `tax_rate` to this list.

What's up with `tax_rate`?

Reply:
We also cache tax rates in Redis with the same TTL as plans and products. Thanks to #10745 , we listen for some tax rate webhooks and update the cache accordingly. If we don't add `tax_rate` to this new `BYPASS_LATEST_FETCH_TYPES`, I imagine we would encounter the same issue if someone tried to add or update a tax rate?

Reply:
Sure, but `expandResource` doesn't read from that cache for tax rates.